nemo/collections/llm/gpt/data/hf_dataset.py

```diff
@@ -157,6 +170,13 @@ def __init__(
         self.use_mcore_sampler = use_mcore_sampler
         self.mcore_dataloader_type = mcore_dataloader_type
 
+    @staticmethod
+    def from_dict(dataset_dict, split, **kwargs):
+        from datasets import Dataset
```

Comment:
Can you move all `datasets` import to the top level? Since there's already `from datasets import load_dataset` at the top level, I think it's better to move everything to the top